baybe/utils/dataframe.py

```diff
@@ -351,6 +351,48 @@ def df_uncorrelated_features(
     return data
 
 
+def df_add_noise_to_degenerate_rows(
```

Comment:
Hmm I see it doesn't change the potential effect ie `add noise to duplicated rows` has the same problem

Is this really important? I dont wanna make it sound like `add noise to rows that were degenerated/duplicated` (because we never degenerated/duplicated them) which is how I would read your suggestions so imo its not really unambiguous either

Reply:
I'll leave the decision to the two of you, but what ever you decide for: shouldn't we drop the `df_` at the beginning, to be consistent with our other utilities like `add_parameter_noise`, `add_fake_measurements` etc?

Reply:
this wouldnt be consistent considering all the other utilities called `df_`

Reply:
Looking at the functions here, it seems to be inconsistent in general - some functions have a `df`, some don't. However, the other `add_....` functions do not have a `df` in front of them, so I still prefer dropping it.

Reply:
Renamed to `add_noise_to_perturb_degenerate_rows`